TombLib/TombLib/Catalogs/TEN Node Catalogs/Revolution.lua

```diff
@@ -178,11 +172,11 @@ end
 -- !Arguments "33, Boolean, Smooth"
 -- !Arguments "NewLine, 65, Boolean, Rotate Object (Y axis when Horizontal selected)"
 -- !Arguments "35, Boolean, Rotate Center Object"
--- !Arguments "NewLine, SoundEffects,{635}",
+-- !Arguments "NewLine, SoundEffects,{635}"

```

Comment:
Why this particular effect ID (635) is used for revolution node by default?

Reply:
Its an appropriate sound for rotating object. Can be set to 0 if needed.